docs/best_practices/repositories.rst

```diff
@@ -24,8 +24,26 @@ repositories in a structure similar to the following:
 The highest level directory contains only a few folders for the major types of
 Galaxy repositories; tools, packages, data managers, and sometimes visualizations and datatypes.
 
-Packages
---------
+ToolShed Repositories
+---------------------
+
+A Github repository many correspond to any number of published ToolShed
+repositories.
+
+Every unrestricted tool shed repository should contain a README_ file -
+named either ``README`` or ``README.txt`` (if plain text) or ``README.rst``
+(if reStructuredText_). A reStructuredText ``README.rst`` is generally
+preferred. For a good example of such a file - please see Peter Cock's NCBI
+Blast+ Suite's `README.rst
+<https://github.com/peterjc/galaxy_blast/blob/master/tools/ncbi_blast_plus/README.rst>`__.
```

Comment:
IUC should transclude an example copy of this at some point. +1 for using @peterjc's awesome reference READMEs.